.github/workflows/run-tests.yaml

```diff
@@ -80,8 +31,7 @@ jobs:
           pip install .[dev]
       - name: Test
         run: |
-          export HF_TAG=${{ hashFiles('setup.cfg', 'src/petals/cli/convert_model.py') }}
-          export MODEL_NAME=bloom-testing/test-bloomd-560m-$HF_TAG
+          export MODEL_NAME=bigscience/bloom-560m
```

Comment:
Loading the entire 560m model takes only 5 sec, so I'd stick to using it.

However, it takes a little more RAM, so we run 4 servers instead of 5 below.